python/lsst/validate/drp/check.py

```diff
@@ -65,6 +67,44 @@ def magNormDiff(cat):
     return normDiff
 
 
+def fitExp(x, y, y_err, deg=2):
+    """Fit an exponential quadratic to x, y, y_err.
+    """
+    fit_params, fit_param_covariance = \
+        curve_fit(expModel, x, y, p0=[1, 0.02, 5], sigma=y_err)
```

Comment:
Thanks for asking this.  I've cleaned things up correctly now.

expModel was defined plot.py.  It should have been (and is now) in check.py

But fitExp was only used in plot.py, so this worked by the time fitExp was called.